### ISSUE TITLE
Add Separate passwords for automatic Encrypted Backups

### ISSUE THREAD

**Issue by FrozenVertx** [beemdevelopment/Aegis#774]
Hi,
I want to request a features for backup as you read in Title. I just want to use some small PIN or Password for app to quickly open it without typing long password. So please add separate password for backup so that if I set any weak password on app then it won't affect backup. Or also adding PIN for password protect app from unwanted access like Authy totally apart from password used for encrypt database is also good if 1st one not possible.
Summing up all I just want to quickly access app without entering long password each time access app.

Thanks

**Comment by alexbakker** [comment]
Duplicate of #121.
